src/program_generator/common.py

```diff
@@ -101,4 +101,11 @@ def _opcodes_dict_push_dup_swap(source, removeds, addeds, parameter=None):
     } for opcode, name, removed, added in zip(opcodes, names, removeds, addeds)
   }
 
-  return new_part
\ No newline at end of file
+  return new_part
+
+# Returns a single MSTORE8 accompanied with pushes, causing pre-allocation of all the memory at program's disposal
+# 6000 - PUSH1 the zero (the byte to store)
+# 630001ffff - PUSH4 128KB-worth-of-bytes minus one byte
```

Comment:
Why `PUSH4`? Why not `PUSH1`?

Reply:
PUSH1 is too small. PUSH4 is too large but :shrug: I think it doesn't matter

Reply:
> makes other opcodes tests less reliable.

That is very much true. We might end up deciding to drop this "preamble" for OPCODEs which don't require it. However, I'm prioritizing OPCODE-fairness over anything else, for now, so all OPCODEs should get as much "same treatment" as possible, on every level. We'll see how things look, and possibly tweak it.

Reply:
It turns out, that cheaper OPCODEs will be very hard to measure with the MSTORE8 pre-amble, so we're going to drop it. I pushed a version which only injects this for those OPCODEs that actually need it to run